benchmarks/benchmark_dht.py

```diff
@@ -1,129 +1,164 @@
 import argparse
 import random
 import time
+import uuid
+
+import asyncio
 
 from tqdm import trange
 
 import hivemind
-from hivemind.moe.server import declare_experts, get_experts
-from hivemind.utils.limits import increase_file_limit
 from hivemind.utils.logging import get_logger, use_hivemind_log_handler
 
 use_hivemind_log_handler("in_root_logger")
 logger = get_logger(__name__)
 
 
-def random_endpoint() -> hivemind.Endpoint:
-    return (
-        f"{random.randint(0, 256)}.{random.randint(0, 256)}.{random.randint(0, 256)}."
-        f"{random.randint(0, 256)}:{random.randint(0, 65535)}"
+async def store_task(peer, key, value, expiration):
+    subkey = uuid.uuid4().hex
+    
+    store_ok = await peer.store(
+        key, subkey=subkey, value=value, 
+        expiration_time=hivemind.get_dht_time() + expiration, return_future=True
     )
 
+    return store_ok
+
+
+async def get_task(peer, key):
```

Comment:
This is essentially a one-liner, perhaps it would be best to inline it into the main coro